src/DynamoCore/Models/AnnotationModel.cs

```diff
@@ -120,7 +121,7 @@ public Double TextBlockHeight
             {
                 textBlockHeight = value;                
                 Y = InitialTop - textBlockHeight;
-                Height = InitialHeight + textBlockHeight;
+                Height = InitialHeight + textBlockHeight - 20;
```

Comment:
It's usually a bad idea to have constants embedded in the code.  Why 20?  Also, it looks like you use it again below.  Consider moving it to a static readonly field.